pagopa/io-app
# feat: [IA-325] Added the accessibility role button to the reset code function

## List of changes proposed in this pull request
- Added the accessibility role "button" to the function used to reset the access code 

## How to test
This is an accessibility improvement so to test it you should execute a screen reader on the smartphone and check the page where you must insert the secure code. The button "reset code" wasn't recognized as a button
